tap_chargebee/client.py

```diff
@@ -12,14 +12,123 @@
 LOGGER = singer.get_logger()
 
 
-class Server4xxError(Exception):
+class ChargebeeError(Exception):
     pass
 
+class Server4xxError(ChargebeeError):
+    pass
+
+class Server429Error(ChargebeeError):
+    pass
+
+class Server5xxError(ChargebeeError):
+    pass
+
+class ChargebeeBadRequestError(Server4xxError):
+    pass
+
+class ChargebeeAuthenticationError(Server4xxError):
+    pass
+
+class ChargebeeForbiddenError(Server4xxError):
+    pass
+
+class ChargebeeNotFoundError(Server4xxError):
+    pass
+
+class ChargebeeMethodNotAllowedError(Server4xxError):
+    pass
+
+class ChargebeeNotProcessedError(Server4xxError):
+    pass
+
+class ChargebeeRateLimitError(Server429Error):
```

Comment:
Passing 429 seems redundant, we can pass 4xx here

Reply:
Removed Server429Error and pass Server4xxError as suggested.